ParallelBuildsMonitorWindowControl.xaml.cs

```diff
@@ -19,6 +66,7 @@ public partial class ParallelBuildsMonitorWindowControl : UserControl
         public ParallelBuildsMonitorWindowControl()
         {
             this.InitializeComponent();
+            this.DataContext = ViewModel.Instance;
```

Comment:
ViewModel as singleton is not a good idea.
However set this.DataContext to ParallelBuildsMonitorWindowCommand is even worse.

I think data should be separated from ParallelBuildsMonitorWindowCommand into DataModel class, then ViewModel should be related to DataModel.
Still have no idea where DataModel should be created (not as singleton) and how passed to command and all controls.
Please submit and advise.